### DISCUSSION TITLE
No npm

### DISCUSSION THREAD

**Discussion by felipecrrmts** [gowebly/gowebly#102] [Ideas]
Is there a possibility of generating the code without npm in project?
Go cmd maybe woulb be a good option

**Comment by koddr** [comment]
@felipecrrmts hi

I didn't really understand why this might be necessary in a project where there is Tailwind and HTMX, which are clearly better to pull up through NPM, and not from CDN in index.html file. 

Please describe in more detail what you want to get from such a change in the Gowebly CLI.

**Reply by felipecrrmts** [reply]
The idea was to use it from cdn instead of npm, lets get my example. The whole point of migrating to htmx was to run away from the npm/javascript world.
But maybe its just me being too pick.

**Reply by pharrisee** [reply]
It could also be the case that the standalone tailwindcss cli is the preferred option, which doesn't require node or npm to be installed.

https://tailwindcss.com/blog/standalone-cli
